### ISSUE TITLE
parallelize validation pipeline

### ISSUE THREAD

**Issue by davidackerman** [janelia-cellmap/dacapo#38]
Wan't prediction, postprocessing and validation all to be blockwise and in parallel.

**Comment by davidackerman** [comment]
closed, duplicate of #57

**Comment by davidackerman** [comment]
closed, duplicate of #57

**Comment by davidackerman** [comment]
duplicate of #57

**Comment by davidackerman** [comment]
duplicate of #57
